src/cookbook/initializing.md

```diff
@@ -71,6 +73,21 @@ location:
     $ echo 'PYTHONPATH=$PYTHONPATH:/usr/local/samba/lib/python2.7/site-packages;export PYTHONPATH' | sudo tee -a /etc/profile.d/samba4-env-build.sh
     $ . /etc/profile.d/samba4-env-build.sh
 
+# Setting up OpenChange dependencies #
```

Comment:
No, nanomsg and libmemcached aren't available in distros (not the good versions).